stock_available_to_promise_release/migrations/16.0.1.0.0/post-migrate.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    _logger.info(
+        "stock_available_to_promise_release: Configure picking types"
+        " to prevent new moves into released pickings"
+    )
+    cr.execute("update stock_picking_type set prevent_new_move_after_release = true")
```

Comment:
This should be in a 16.0.2.0.0 migration script as this is also required to run on instances having already the module running in 16.0